core/core.go

```diff
@@ -538,6 +539,7 @@ func (app *App) AddOrders(signedOrdersRaw []*json.RawMessage) (*zeroex.Validatio
 		Accepted: []*zeroex.AcceptedOrderInfo{},
 		Rejected: []*zeroex.RejectedOrderInfo{},
 	}
+	orderHashesSeen := map[common.Hash]interface{}{}
```

Comment:
```suggestion
	orderHashesSeen := map[common.Hash]struct{}{}
```

Usually, you want to use an empty struct (`struct{}`) for the value type if you don't intend to use it. It is one of few data structures (maybe the only one?) that doesn't use any bytes of memory. The empty interface (`interface{}`) *does* use up some memory.